packages/containerd-2.0/containerd.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=containerd container runtime
+Documentation=https://containerd.io
+After=network-online.target configured.target
+Wants=network-online.target configured.target
+
+[Service]
+Slice=runtime.slice
+Environment=CONTAINERD_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE=1
+EnvironmentFile=/etc/network/proxy.env
+EnvironmentFile=-/etc/containerd/nvidia.env
```

Comment:
`nvidia.env` isn't needed anymore, (it was removed here: https://github.com/bottlerocket-os/bottlerocket-kernel-kit/pull/84 )